src/appleseed/renderer/modeling/object/object.cpp

```diff
@@ -107,4 +107,8 @@ void Object::on_frame_end(
     ConnectableEntity::on_frame_end(project, parent);
 }
 
+void Object::rasterize(ObjectRasterizer& rasterizer) const
+{
```

Comment:
assert(!"Not implemented")? Or even throw

Reply:
By default the method does nothing on purpose: most objects don't know how to rasterize themselves yet but we don't want this to be blocking.